### ISSUE TITLE
get-config fails to retrieve config data

### ISSUE THREAD

**Issue by Raaghavi** [OpenClovis/OpenYuma#24]
Hi,

Initially I tried to do an edit-config operation on the candidate datastore through yangcli & I was able to retrieve the data through get-config. After I invoked a commit rpc, I could see that the get-config data either does not have any instance that is set through the previous edit or it displays some random values against each leaf/container.

Could you please explain the cause of this & suggest me the steps to resolve this issue? Thanks in advance.

Regards,
Raaghavi

**Comment by hoangle** [comment]
What is  default database to edit on netconf server?
If default is 'candidate', you have to do copy from running to candidate before starting to edit configuration.
Take a look at doc: OpenYuma/netconf/doc/yuma_docs/openyuma-dev-manual.odt (Chapter 5.2) to know how to change this target.

**Comment by Raaghavi** [comment]
I am using candidate as the default database on the netconf server.
Both the candidate & the running datastores are empty when I start the netconf server. I tried the method you suggested by copying the running configuration into candidate datastore before applying an edit. But still I am facing the same issue.
I also tried by making the running datastore as the default database for the netconf server. Here I made an edit into the running datastore directly without using the commit & found no difference. get-config didn't retrieve the data I posted.
